rpcserver.go

```diff
@@ -435,14 +436,14 @@ func rpcMiscError(message string) *dcrjson.RPCError {
 type workState struct {
 	sync.Mutex
 	prevHash     *chainhash.Hash
-	templatePool map[[merkleRootPairSize]byte]*wire.MsgBlock
+	templatePool map[[mining.MerkleRootPairSize]byte]*wire.MsgBlock
```

Comment:
`merkleRootPairSize` is only used in the RPC server.  It should _NOT_ be in the mining module.  The constant should just be moved into the `rpcserver.go` file where it belongs.  That will also remove the need to change a bunch of these.